code/game/turfs/space/space.dm

```diff
@@ -61,7 +61,7 @@
 	var/list/y_arr
 
 	if(src.x <= 1)
```

Comment:
This is probably where your null.x runtimes are coming from, all the code relating to meteors should be removed from this and handled in your meteor code.